src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/executors/spark/SparkEmrClusterServiceImpl.scala

```diff
@@ -62,8 +62,8 @@ class SparkEmrClusterServiceImpl @Inject()(
 
       val response = emr.addJobFlowSteps(jobFlowStepsRequest)
       val stepId = response.getStepIds.asScala.headOption
-      logger.info(s"Added jobFlowStepsRequest ${jobFlowStepsRequest} for executorId ${id} and stepId $stepId}")
-      logger.info(response.toString)
+      logger.debug(s"Added jobFlowStepsRequest ${jobFlowStepsRequest} for executorId ${id} and stepId $stepId}")
+      logger.debug(response.toString)
```

Comment:
I'd like to keep this as `.info` because it's only during submission, so should not cause excessive logging

Reply:
Done